samples/snippets/pom.xml

```diff
@@ -36,13 +36,18 @@
       </dependency>
     </dependencies>
   </dependencyManagement>
+  <!-- [END bigqueryconnection_install_with_bom] -->
 
+  <!-- We cannot pull beta clients from libraries-bom, specify as below for now -->
+  <!-- [START bigqueryconnection_install_without_bom] -->
```

Comment:
I don't think we want either snippet in here. The install_without_bom is in samples/install-without-bom/pom.xml. The `install_with_bom` snippet will still appear in the README if it's present (and it will be incomplete and incorrect).

If there's no install_with_bom snippet, the README will show the existing install_without_bom snippet

Reply:
done